src/main/java/org/isf/lab/model/Laboratory.java

```diff
@@ -124,7 +125,7 @@ public Laboratory() { }
 	public Laboratory(Exam aExam,GregorianCalendar aDate,String aResult,
 			String aNote, Patient aPatId, String aPatName){
 		exam=aExam;
-		registrationDate=aDate;
+		labDate=aDate;
```

Comment:
Nitpick: the formatting in this and the next method could use some work.

Reply:
Fixed, thanks!